doc/source/release-notes/Percona-Server-5.6.28-76.1.rst

```diff
@@ -0,0 +1,28 @@
+.. rn:: 5.6.28-76.1
+
+==============================
+ |Percona Server| 5.6.28-76.1 
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.28-76.1 on January 12th, 2016 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.28-76.1/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.28 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-28.html>`_, including all the bug fixes in it, |Percona Server| 5.6.28-76.1 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.28-76.1 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.28-76.1>`_.
+
+Bugs Fixed
+==========
+ 
+ Clustering index could not be created on partitioned |TokuDB| table. Bug fixed :bug:`1527730`.
```

Comment:
Clustering secondary indexes could not be created on ... Maybe there is a bug ref in DB JIRA too?